tbx/project_styleguide/templates/patterns/_pattern_library_only/streamfield/division_story_container.html

```diff
@@ -0,0 +1 @@
+{% include "patterns/molecules/streamfield/blocks/introduction_with_images_block.html" %}
```

Comment:
q: should this not have four_photo_collage and partners_block too?

Reply:
That's a good shout.

My initial thought was to not have the four photo collage block because it was the alternative for the introduction, but it can't hurt to have it. Not sure how I missed adding the partners block too.

Done in [7d82a84](https://github.com/torchbox/torchbox.com/pull/324/commits/7d82a84d16e8c3a972540e2f49d0f5d2c8c649c9)